src/app/components/local/store-details/store-details.component.ts

```diff
@@ -148,4 +169,30 @@ export class StoreDetailsComponent implements OnInit {
       return a + b.price;
     }, 0);
   }
+
+  plusSlides(n:any) {
```

Comment:
I would suggest to rename the method names like "moveSlideLeft", "moveSlideRight" & "moveSlide". PlusSlide implies a wrong intention. ( Think if you can get a better names)